Description: Build tm_posix.cpp on kFreeBSD and Hurd to fix a crash
 because of an undefined symbol.
Author: Felix Geyer <fgeyer@debian.org>
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/683933

--- qtcreator-2.5.0.orig/src/libs/3rdparty/botan/src/src.pro
+++ qtcreator-2.5.0/src/libs/3rdparty/botan/src/src.pro
@@ -282,7 +282,7 @@ unix {
            timer/gettimeofday/tm_unix.h
 }
 
-linux*-g++*|freebsd*-g++* {
+linux*-g++*|freebsd*-g++*|glibc*-g++*|hurd*-g++* {
     HEADERS += timer/posix_rt/tm_posix.h
 }
 
@@ -566,7 +566,7 @@ unix {
            timer/gettimeofday/tm_unix.cpp
 }
 
-linux*|freebsd* {
+linux*|freebsd*|glibc*|hurd* {
     SOURCES += timer/posix_rt/tm_posix.cpp
 
     LIBS += -lrt
